internal/fs/caching_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	"path"
```

Comment:
can we also write tests for multi-bucket (dynamic mount) here ?

Reply:
Added. PTAL.